src/components/EmbeddedCheckoutProvider.tsx

```diff
@@ -171,6 +173,15 @@ export const EmbeddedCheckoutProvider: FunctionComponent<PropsWithChildren<
       return;
     }
 
+    if (
+      options.clientSecret === undefined ||
```

Comment:
```suggestion
      options.clientSecret === undefined &&
```

hmm, should this be `&&`? 🤔 
according to the test description `"errors if both clientSecret and fetchClientSecret are undefined"`

Reply:
Ahh, yes! Thank you for the catch. I'll add another test to cover this.